Raven.Client.Lightweight/Document/InMemoryDocumentSessionOperations.cs

```diff
@@ -1210,7 +1210,7 @@ protected bool EntityChanged(object entity, DocumentMetadata documentMetadata, I
                 documentMetadata.Metadata.Value<bool>(Constants.RavenReadOnly))
                 return false;
 
-            var newObj = EntityToJson.ConvertEntityToJson(documentMetadata.Key, entity, documentMetadata.Metadata);
+            var newObj = EntityToJson.ConvertEntityToJson(documentMetadata.Key, entity, documentMetadata.Metadata, executeListeners: false);
```

Comment:
This may cause us to send changes to the database that we shouldn't. 
Because the listener wasn't called to mutate the data

Reply:
@grisha-kotler ?